cfgov/v1/util/queries.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import ref
+
+from ..models.learn_page import AbstractFilterPage
```

Comment:
Possibly nitpicky style thing, but PEP8 [recommends](https://www.python.org/dev/peps/pep-0008/#imports) absolute imports over relative imports. [This](http://programmers.stackexchange.com/questions/159503/whats-wrong-with-relative-imports-in-python/159505#159505) is also a good explanation of the downsides of relative imports.